crates/steel-core/src/parser/expander.rs

```diff
@@ -391,6 +374,7 @@ impl MacroCase {
 
 #[derive(Clone, PartialEq, Serialize, Deserialize)]
 pub enum MacroPattern {
+    Rest(Box<MacroPattern>),
```

Comment:
:information_source: A new pattern type for anything in the `cdr` of a cons list.